langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -1041,6 +1041,11 @@ function testSort10() {
 
     assertValueEquality(sortedArr3.toString(), "[0,1,10,2,3,6]");
     assertValueEquality(sortedArr3, arr);
+
+    int[] sortedArr4 = arr.sort(array:DESCENDING, (i) => i.toString());
+
+    assertValueEquality(sortedArr4.toString(), "[6,3,2,10,1,0]");
```

Comment:
Instead of using toString shall we assert the value itself? We lose type information when we assert using toString.